cc-ansible

```diff
@@ -325,4 +325,11 @@ if [[ "${POSARGS[*]}" =~ post-deploy ]]; then
 fi
 kolla_args+=("${POSARGS[@]}")
 
+#load tags to skip
+ANSIBLE_SKIP_TAGS="$(grep -v "^#" ./kolla-skip-tags | paste -sd ',')"
+if [[ "${ANSIBLE_SKIP_TAGS}" ]]; then
+  kolla_args+=("--skip-tags")
+  kolla_args+=("${ANSIBLE_SKIP_TAGS[@]}")
```

Comment:
Do you need the `[@]`?

Reply:
I need to check, that's left over from when it was getting interpreted as an array.